cmd/manager/aggregator.go

```diff
@@ -336,7 +340,9 @@ func getCheckResultLabels(pr *utils.ParseResult, resultLabels map[string]string,
 func getCheckResultAnnotations(cr *compv1alpha1.ComplianceCheckResult, resultAnnotations map[string]string) map[string]string {
 	annotations := make(map[string]string)
 	annotations[compv1alpha1.ComplianceCheckResultRuleAnnotation] = cr.IDToDNSFriendlyName()
-
+	if len(cr.ValueUsed) > 0 {
+		annotations[compv1alpha1.ComplianceCheckResultValueAnnotation] = strings.Join(cr.ValueUsed, ",")
+	}
```

Comment:
I'm not sure annotation is needed for this if we're adding the ValuesUsed key to the CRD itself. Is there any specific use-case in mind for adding this annotation? If we want this information to be on the metadata somehow, I would suggest skipping the label and sticking only to the annotation. But then again, I still think it's enough if we have it on the `ValuesUsed` key on the CRD.

Reply:
I have changed the `ValueUsed` to `ValuesUsed`, and deleted the annotations, I am going to keep the label so that it will be easier to search if a `ComplianceCheckResult` has used variables